automation/test.sh

```diff
@@ -47,12 +47,18 @@ elif [[ $TARGET =~ cnao ]]; then
   export KUBEVIRT_DEPLOY_CDI=false
 elif [[ $TARGET =~ sig-network ]]; then
   export KUBEVIRT_WITH_CNAO=true
-  export KUBEVIRT_PROVIDER=${TARGET/-sig-network/}
-  export KUBEVIRT_DEPLOY_ISTIO=true
   export KUBEVIRT_DEPLOY_CDI=false
-  if [[ $TARGET =~ k8s-1\.1.* ]]; then
-    export KUBEVIRT_DEPLOY_ISTIO=false
+  # FIXME: https://github.com/kubevirt/kubevirt/issues/9158
+  if [[ $TARGET =~ 1\.26 ]]; then
+    if [[ $TARGET =~ istio ]]; then
+      export KUBEVIRT_DEPLOY_ISTIO=true
+    else
+      export KUBEVIRT_DEPLOY_ISTIO=false
+    fi
+  else
+    export KUBEVIRT_DEPLOY_ISTIO=true
   fi
+  export KUBEVIRT_PROVIDER=${TARGET/-sig-network*/}
```

Comment:
we still need to remove `-istio`
else this will break
[KUBEVIRT_PROVIDER](https://github.com/kubevirt/project-infra/pull/2592/files#diff-26f7cd0389085c1da47f919a02c4dbeca5ddd96ca00d8e49c0862f698475aaf9R2454)